common/src/main/java/io/druid/common/guava/GuavaUtils.java

```diff
@@ -95,4 +97,11 @@ public InputStreamReader getInput() throws IOException
       }
     };
   }
+
+  public static Long tryParseLong(String string)
```

Comment:
Is this intended `@Nullable` on the input argument? There are checks for things that call this for `nullToEmpty`, which are probably not needed.